Swag/Paths/FAQ/FAQ.yaml

```diff
@@ -0,0 +1,102 @@
+# GET /faq
+AllFAQs:
+  tags:
+    - FAQ
+  summary: Returns list of FAQs
+  operationId: getAllFAQs
+  description: This end point returns the list/collection of frequently asked questions(with answers) on/about CodingGarden Community.
```

Comment:
This should be pharsed differently, most question are directed to CJ (not the codinggarden communnity)

Reply:
Yes, but that for the moment right? As the community grows, it would get more questions for the community itself. Also, this is an internal Doc for contributors. Not something that 's public-facing.. What do you think?

Reply:
Not that important, just add something like "and CJ" and the end and it will be fine.

Reply:
I literally did that 😬